Library/Koala/Koala.swift

```diff
@@ -1769,14 +1774,14 @@ private func properties(category category: KsApi.Category) -> [String:AnyObject]
     .withAllValuesFrom(parentProperties.prefixedKeys("parent_"))
 }
 
-private func properties(shareContext shareContext: ShareContext,
-                                     loggedInUser: User?,
-                                     shareActivityType: String? = nil) -> [String:AnyObject] {
+private func properties(shareContext: ShareContext,
+                        loggedInUser: User?,
+                        shareActivityType: UIActivityType? = nil) -> [String:Any] {
```

Comment:
UIActivityType as a struct can be extended, but enums can't. Didn't know about RawRepresentable to kind of turn it back into an enum though, that's cool!